src/actionlib/SimpleActionServer.js

```diff
@@ -185,6 +185,29 @@ SimpleActionServer.prototype.sendFeedback = function(feedback2) {
     this.feedbackPublisher.publish(feedbackMessage);
 };
 
+/**
+*  Set action state to aborted and return to client
+*/
+
+SimpleActionServer.prototype.setAborted = function(result2) {
```

Comment:
`result2`?

Reply:
copied from upstream but can change the name. `result2` is used all over lol.